### DISCUSSION TITLE
Custom dictionary path

### DISCUSSION THREAD

**Discussion by ookii-tsuki** [Cutano/Kawazu#14] [Ideas]
Hey Cutano,
Wouldn't it be better if we can specify a custom dictionary path, for example, we can provide the dictionary path in the constructor parameters like this ` using var conv = new KawazuConverter(@"C:\Program Files (x86)\MeCab\dic\ipadic");` and if nothing is provided, it should use the default path

**Comment by Cutano** [comment]
That's a good idea, but I'm wondering if that could lead to a problem when you build the app, and someone else executes it with no dic files in that path.
Is there any way to integrate dic files into the app being built?

**Reply by ookii-tsuki** [reply]
We can just throw an error that the dic files don't exist and the developer can handle them by sending a message to the user that they should add those files or it can automatically copy the files to that directory or things like that but I'm not sure if you can integrate the dic files into the app, I'm using Unity in which I can put them in the resources folder then extract them.

**Reply by Cutano** [reply]
Yeah, that will be better.
Definitely better than the dummy fixed internal path we are using now 🤔

**Reply by ookii-tsuki** [reply]
Yep exactly
